Updates to CoC and related documents (#747)
writethedocs/www: docs/blog/new-community-coc-2019.rst

.. post:: Feb 1, 2019
   :tags: community, code of conduct, announcements

Improved Community Code of Conduct
==================================

> As you may know, we aim to make Write the Docs an incredibly diverse, friendly, and inclusive community. As part of that effort, we have a community :doc:`/code-of-conduct`. We have refactored and improved our Code of Conduct, based on lessons in our own and other communities, and how our community has developed in recent years.

Reviewer: ```suggestion
We aim to make Write the Docs an incredibly diverse, friendly, and inclusive community. As part of that effort, we have a community :doc:`/code-of-conduct`. We have refactored and improved our Code of Conduct, based on how our community has developed and lessons we've learned from other communities.```

Reviewer: Not sure about that final sentence.

Author: I'm not sure about it either. It should be a positive thing, what we had wasn't bad, but it can be better. Also, it shouldn't make it seem like anything bad happened to cause this change. But it may be good to have some brief motivation for change.

The most significant changes we've made are:

> * The CoC now explains the fundamental goals we have with a CoC.

Reviewer: I'd prefer all of those points to be more like "we've clarified the fundamental CoC goal. We've added clarification.... 

What do you think?

Author: Yes, it's more active language. But I was also trying to avoid a lot of repetitive language.

Reviewer: Yeah, one way around that is to split the list:

We've added the following
* A
* B

And we've updated:
*C
*D

* Clarification was added on cases where the CoC can apply during conferences, even for events not on the conference agenda.
* A short guideline on reporting was added. Community members are now more strongly encouraged to report incidents, even if they are unsure whether it is a violation or not. This is also reflected in the CoC in other places.
* Clarification was added on who to report to in which spaces, how a report can be escalated, and which people are in which response team.
* The list of examples of harassment and exclusionary behaviour is expanded, to provide more clarity on what can fall under this.
* The :doc:`/code-of-conduct-response` was restructured to provide better guidance to people in CoC response teams in different community spaces, how to work as a team, deal with conflicts of interest, handling public statements, and how to decide on a response.

> * An explicit policy was added that response team members are not to involve venue security or law enforcement without consent of the affected person(s), except if there is no other reasonable option given significant safety risks.

Reviewer: ```suggestion 
* An explicit policy was added regarding venue security or law enforcement involvement. They should not be involved without consent of the affected person(s), except if there is no other reasonable option given significant safety risks.
```
